### PR TITLE
Add optional value to ExplicitNull and ServerTimestamp

### PR DESCRIPTION
**Pull request by wu-hui**

### CODE REVIEW

File: Firestore/Swift/Source/Codable/ExplicitNull.swift [hunk 1]
```diff
@@ -39,7 +39,7 @@ public enum ExplicitNull<Wrapped> {
   }
 
   /// Get the `Optional` representation of `ExplicitNull`.
```
Comment (wilhuff) [inline Firestore/Swift/Source/Codable/ExplicitNull.swift]: Comments should be phrased with an implicit "This method ..."

What do you think of rephasing this as

> Returns this value as an `Optional<Wrapped>`?

File: Firestore/Swift/Source/Codable/FieldValue+Encodable.swift [hunk 2]
```diff
@@ -49,6 +49,16 @@ public enum ServerTimestamp: Codable, Equatable {
   /// value to `stamp`.
   case resolved(Timestamp)
 
+  /// Get the `Optional` representation of `ServerTimestamp`.
```
Comment (wilhuff) [inline Firestore/Swift/Source/Codable/FieldValue+Encodable.swift]: Similar concern about phrasing here. What do you think of this?

> Returns this value as an `Optional<Timestamp>`.
> 
> If the server timestamp is still pending, the returned optional will be `.none`. Once resolved, the returned optional will be `.some` with the resolved timestamp.

File: Firestore/Swift/Source/Codable/ExplicitNull.swift (hunk 1 above)
Comment (wu-hui) [reply Firestore/Swift/Source/Codable/ExplicitNull.swift]: Better, done.

File: Firestore/Swift/Source/Codable/FieldValue+Encodable.swift (hunk 2 above)
Comment (wu-hui) [reply Firestore/Swift/Source/Codable/FieldValue+Encodable.swift]: Done.

**Pull request merged**